schema-extended/Version.json

```diff
@@ -0,0 +1,266 @@
+      "$ref": "Common.json#/definitions/titles"
+    },
+    "abstracts": {
+      "$ref": "Common.json#/definitions/abstracts"
+    },
+    "legacy_authors": {
+      "title": "Legacy authors",
+      "description": "The legacy authors field, which is an unstructured string.",
+      "type": "string",
+      "minLength": 1,
+      "examples": [
+        "J. Doe (Cornell University), M. Smith (Stanford University), F. Bar"
+      ]
+    },
+    "authors": {
+      "title": "Authors",
+      "type": "array",
+      "items": {
+        "$ref": "Person.json"
+      },
+      "minItems": 1
+    },
+    "collaborations": {
+      "title": "Collaborations",
+      "description": "Collaborations associated with this e-print. Should not include 'Collaboration' suffix.",
+      "type": "array",
+      "items": {
+        "$ref": "Common.json#/definitions/collaboration"
+      },
+      "examples": [
+        "ATLAS",
+        "BABAR",
+        "L3",
+        "ZEUS"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "funding": {
+      "$ref": "Funding.json"
+    },
+    "comments": {
+      "title": "Comments",
+      "description": "Comments from authors or arXiv administrators.",
+      "type": "object",
+      "properties": {
+        "author_comments": {
+          "title": "Author comments",
+          "description": "Comments from authors.",
+          "type": "array",
+          "items": {
+            "$ref": "Common.json#/definitions/comment"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "administrator_comments": {
+          "title": "Administrator comments",
+          "description": "Comments from arXiv administrators",
+          "type": "array",
+          "items": {
+            "$ref": "Common.json#/definitions/comment"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "legacy_comments": {
+          "description": "Unstructured comments from authors and/or arXiv administrators.",
+          "$ref": "Common.json#/definitions/comment"
+        }
+      }
+    },
+    "journal_ref": {
+      "title": "Journal reference",
+      "description": "The full bibliographic reference if the article has already appeared in a journal, proceedings, etc.",
+      "$ref": "Common.json#/definitions/journal_ref"
+    },
+    "publisher_dois": {
+      "title": "List of publisher-provided DOIs",
+      "description": "The publisher DOI(s).",
+      "type": "array",
+      "items": {
+        "$ref": "Common.json#/definitions/doi"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "submitter_dois": {
+      "title": "List of submitter-provided DOIs",
+      "description": "The submitter DOI(s).",
+      "type": "array",
+      "items": {
+        "$ref": "Common.json#/definitions/doi"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "report_numbers": {
```

Comment:
I remember discussing in one of the WG sessions how to handle the case of multiple report numbers for one record. IMO it would be best to have a separate field for each individual report number (e.g. instead of submitters listing multiple report numbers in their own unstandardised way and separating them with comma/semicolon/backslash/etc. in one freetext field, it would be better to be able to add multiple individual report number fields). Is this something that is foreseen? Or is this rather a question of the submission form?